consensus/enclave/impl/src/lib.rs

```diff
@@ -177,20 +180,25 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         peer_self_id: &ResponderId,
         client_self_id: &ResponderId,
         sealed_key: &Option<SealedBlockSigningKey>,
-        minimum_fee: Option<u64>,
+        minimum_fees: Option<BTreeMap<TokenId, u64>>,
     ) -> Result<(SealedBlockSigningKey, Vec<String>)> {
-        // Inject the fee into the peer ResponderId
-        let peer_self_id = if let Some(fee) = minimum_fee {
-            let str = format!("{}-{}", &peer_self_id.0, fee);
-            ResponderId(str)
-        } else {
-            peer_self_id.clone()
-        };
+        // If we are provided with the optional fee map, we must have >0 MOB minimum
```

Comment:
Would we want to iterate through all the fees in the token map to make sure they're all >0?

Reply:
That's a possibility. We haven't established yet the fee structure for other token ids, but I agree it does seem like a safe bet to want them to be positive if they're there.
I'm now thinking that maybe the right place for this logic might be in the fee map itself. Thoughts?

Reply:
Decided to implement this.